### ISSUE TITLE
Licence confusion

### ISSUE THREAD

**Issue by andybak** [dizzy2003/TextureWang#9]
> While the source is public, the code itself is not open source.

Well - some of it is as Node_Editor_Framework is MIT licensed!

However - there's something to stop you asserting copyright on your modifications. It would make it clearer if you added a simple copyright message to licence.md

> Its simply my intent to stop others from being able to package up this code or derivative and sell it on the Unity Asset Store or any other store.

This is the bit that puzzles me. How does publishing the code on Github help you do that?

**Comment by dizzy2003** [comment]
I agree its confusing :) <I will edit the part about Node_Editor_Framework>and I havent had much time to look into licenses. yes part of the code is mit licensed, and thats in its own sub folder and is only slightly modified since I upgraded to latest version.

Why is it on gitHub at all? To make the tool freely accessible to download modify and use the tool.

**Comment by andybak** [comment]
> Why is it on gitHub at all? To make the tool freely accessible to download modify and use the tool.

Right. So you're happy for people to make use of it but not to republish it as their own? The tricky bit is about "derivatives". I'm assuming if someone took this and made amazing improvements to it then you'd be happy for them to republish it with appropriate credits? 

How about if I click the nice green "fork" button at the top of the main repo page? I just copied the code under my own Github account.

Licences are a tricky business. I suspect you probably want something along the lines of the GPL for this -  it won't stop commercial use but it probably prevents your hard work from being exploited in ways that you care about. Some people attempt to enforce non-commercial usage for code but it's problematic for several reasons. (I think it's fairly problematic for Creative Commons too but at least the concept of reuse is simpler outside of programming).
